src/main/resources/templates/index.html

```diff
@@ -15,8 +16,10 @@
         <h1 id="logo">AMAZIN</h1>
         <p>The place where your books reside</p>
     </div>
-    <div th:replace=" ~{fragments/_bookBrowser.html :: dataTable} ">
-    </div>
+    <div th:replace=" ~{fragments/_bookBrowser.html :: dataTable} "></div>
```

Comment:
I had to look this up since I didn't know the syntax, so maybe this will help someone who has the same question: the ":: dataTable" portion of the "th:replace" statement specifies the fragment we which to use within the "fragments/_bookBrowser.html" since Thymeleaf supports multiple fragments in a single html file.